### PR TITLE
Gravity project added  issue #795

### LINKED ISSUE #795: Title: Gravity Simulator
### Is your feature request related to a problem? Please describe.

A desktop application to play with weight and gravity values

### Describe the solution you'd like.

It helps visualize how much you will weigh on another planet

### Describe alternatives you've considered.

User friendly

### Add any other context or screenshots about the feature request here.

_No response_

### PR DESCRIPTION
**Pull request by programmingninjas**
### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

Yes

### Description

It's a gravity simulator where you can compare weights on different planets by setting values of g. Made in python and comes under desktop application category

Fixes #795 

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Related issue is #795

### CODE REVIEW

**Comment by Kushal997-das** [comment]
@programmingninjas Fill the from correctly .

> ### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?
> (Write your answer here.)
> 
> ### Description
> (Write your answer here.)
> 
> Fixes #795
> 
> ### Checklist
> * [ ]  I've read the contribution guidelines.
> * [ ]  I've checked the issue list before deciding what to submit.
> * [ ]  I've edited the `README.md` and link to my code.
> 
> ## Related Issues or Pull Requests
> (Write your answer here.)

Write all the answer that has been asked .

**Comment by programmingninjas** [comment]
@Kushal997-das sorry I updated the form please review now

File: Desktop Application/Basic/Python/Gravity/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+2. set values of g of planets you want to compare
+
+## Output
+1. Default weight on earth is taken 80
+2. Comparing Moon and Earth<br/>
+![op](https://raw.githubusercontent.com/programmingninjas/Project-Guidance/main/Desktop%20Application/Basic/Python/Gravity/demo.png)
+
+
+
+
```
Comment (Kushal997-das) [inline Desktop Application/Basic/Python/Gravity/README.md:15]: Images are not loading Please check.

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
Fix the issues .

**Comment by programmingninjas** [comment]
I changed the image link by adding your username instead of mine @Kushal997-das

File: Desktop Application/Basic/Python/Gravity/README.md (hunk 1 above)
Comment (programmingninjas) [reply Desktop Application/Basic/Python/Gravity/README.md]: link updated

**Review by programmingninjas — COMMENTED** [review]
link updated

**Comment by Kushal997-das** [comment]
> I changed the image link by adding your username instead of mine @Kushal997-das

NO man this way you can't display the image . Try to fix it.

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
Images are not working .

**Comment by programmingninjas** [comment]
@Kushal997-das kindly check again now I uploaded the image in README.md

**Review by Kushal997-das — APPROVED** [review]
LGTM 💯

**Pull request merged**